eslintls installation fails

To reproduce:
1. Uninstall eslintls using `LspUninstall eslintls`
2. Quit Neovim
3. Install it back using `LspInstall eslintls`
4. Get an erroneous output

![image](https://user-images.githubusercontent.com/26769888/114321817-a6ab0400-9ad1-11eb-9f7f-54ea70d15233.png)

<details><summary><b>LspInfo output</b></summary>

Configured servers: cssls, emmet_ls, tsserver, html, eslintls
Neovim logs at: /home/zlksnk/.cache/nvim/lsp.log

1 client(s) attached to this buffer: tsserver

  Client: tsserver (id 1)
  	root:      /home/zlksnk/personal/s
  	filetypes: javascript, javascriptreact, javascript.jsx, typescript, typescriptreact, typescript.tsx
  	cmd:       /home/zlksnk/.local/share/nvim/lsp_servers/tsserver/node_modules/.bin/typescript-language-server --stdio
  	


1 active client(s): 

  Client: tsserver (id 1)
  	root:      /home/zlksnk/personal/s
  	filetypes: javascript, javascriptreact, javascript.jsx, typescript, typescriptreact, typescript.tsx
  	cmd:       /home/zlksnk/.local/share/nvim/lsp_servers/tsserver/node_modules/.bin/typescript-language-server --stdio
  	


Clients that match the filetype javascript:

Config: emmet_ls
	filetype: No filetypes defined, please define filetypes in setup().
  
  Config: tsserver
  	cmd:               /home/zlksnk/.local/share/nvim/lsp_servers/tsserver/node_modules/.bin/typescript-language-server --stdio
  	cmd is executable: True
  	identified root:   /home/zlksnk/personal/s
  	custom handlers:   

Config: eslintls
	filetype: No filetypes defined, please define filetypes in setup().

</details>

**Version info**:
Plugin commit: d10094cc9854c4616075c900dc47d4487b1d4d5d
Nvim version: NVIM v0.5.0-dev+1233-g82ac44d01 \ Build type: RelWithDebInfo
Node.js version: 14.16.0
NPM version: 6.14.11
Operating system: Ubuntu 20.04.2 LTS on Windows 10 x86_64

Additional information: NPM 6 uses v1 lockfile but eslintls uses v2.

`npx tsc` fails with that error:

```
error TS6047: Argument for '--target' option must be 'ES3', 'ES5', or 'ES6'.
error TS5023: Unknown compiler option 'moduleResolution'.
error TS5023: Unknown compiler option 'lib'.
```

`npx tsc -v` output:

```
message TS6029: Version 1.5.3
```

I'm on WSL (Ubuntu 20.04.2 LTS on Windows 10 x86_64). I can reproduce the same issue on an another machine running Ubuntu.

--- a/lua/nvim-lsp-installer/servers/eslintls.lua
+++ b/lua/nvim-lsp-installer/servers/eslintls.lua
@@ -64,7 +64,7 @@ git clone https://github.com/microsoft/vscode-eslint .;
 npm install;
 cd server;
 npm install;
-npx tsc;
+../node_modules/.bin/tsc;
 ]]
 
 return server.Server:new {

